include/libcgroup/groups.h

```diff
@@ -23,6 +23,13 @@ enum cg_version_t {
 	CGROUP_DISK = 0xFF,
 };
 
+enum cg_setup_mode_t {
+	CGROUP_UNKNOWN = 0,
```

Comment:
To avoid confusion and collisions with `enum cg_version_t` would it be better to name these something like:
```
CGROUP_MODE_UNK,
CGROUP_MODE_LEGACY,
CGROUP_MODE_HYBRID,
CGROUP_MODE_UNFIED,
```

Reply:
@drakenclimber Apology for the late reply was caught up with other projects. The suggested renames look better will change them.